src/en/getting-started-jaas.md

```diff
@@ -0,0 +1,127 @@
+Title: Getting started with Juju as a Service
+
+# Getting started with Juju as a Service
+
+Juju as a Service (JAAS) is the fastest and easiest way to quickly get started
+modeling and deploying your cloud-based applications. When you use JAAS,
+Canonical manages the Juju infrastructure. This frees you to concentrate on
+your software and solutions. With JAAS you can deploy, configure, and operate
+your applications on AWS, GCE, and Azure.
```

Comment:
public clouds like ... 
and I would write out the full names, and probably add links